Formula/wxmac-stable.rb

```diff
@@ -0,0 +1,70 @@
+class WxmacStable < Formula
+  desc "Cross-platform C++ GUI toolkit (wxWidgets for macOS) - Stable Release"
+  homepage "https://www.wxwidgets.org"
+  url "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2"
+  sha256 "440f6e73cf5afb2cbf9af10cec8da6cdd3d3998d527598a53db87099524ac807"
+  license "wxWindows"
```

Comment:
This fails the audit because `wxWindows` is deprecated. But https://github.com/wxWidgets/wxWidgets/blob/master/docs/licence.txt 🤷